app/services/doc_auth_router.rb

```diff
@@ -199,6 +199,17 @@ def self.client(warn_notifier: nil)
   # rubocop:enable Layout/LineLength
 
   def self.doc_auth_vendor
+    if IdentityConfig.store.doc_auth_vendor_randomize
+      target_percent = IdentityConfig.store.doc_auth_vendor_randomize_percent
+
+      target_percent = target_percent > 100 ? 100 : target_percent
+      target_percent = target_percent < 0 ? 0 : target_percent
```

Comment:
I think we can simplify thanks to our friend [Comparable#clamp](https://ruby-doc.org/core-2.4.0/Comparable.html#method-i-clamp)

```suggestion
      target_percent = IdentityConfig.store.doc_auth_vendor_randomize_percent.clamp(0, 100)
```

Reply:
I forgot about clamp. Thanks.